modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommercePriceEntryLocalServiceImpl.java

```diff
@@ -583,6 +583,26 @@ public CommercePriceEntry getInstanceBaseCommercePriceEntry(
 		return commercePriceEntries.get(0);
 	}
 
+	@Override
+	public List<CommercePriceEntry> getInstanceCommercePriceEntries(
```

Comment:
is _Instance_ needed? (I know that is like this everywhere in this file but...)

Reply:
I had bad experience on already existing methods 😄

Reply:
IDK it's just the name of the previous method

Reply:
I've just noticed that we have this:
@Override
	public CommercePriceEntry getInstanceBaseCommercePriceEntry(
		String cpInstanceUuid, String priceListType, String unitOfMeasureKey) {

It looks like it has a meaning I'll try to see what Brian thinks